Charon/VirtualFile.py

```diff
@@ -1,24 +1,24 @@
 # Copyright (c) 2018 Ultimaker B.V.
 # libCharon is released under the terms of the LGPLv3 or higher.
+import os
 
-import os.path #To get the file extension from a path.
+from Charon.FileInterface import FileInterface  # The interface we're implementing.
+from Charon.OpenMode import OpenMode  #T o open local files with the selected open mode.
+# The supported file types.
+from Charon.filetypes.OpenPackagingConvention import OpenPackagingConvention
+from Charon.filetypes.GCodeFile import GCodeFile
+from Charon.filetypes.GCodeGzFile import GCodeGzFile
 
-from .FileInterface import FileInterface #The interface we're implementing.
-from .OpenMode import OpenMode #To open local files with the selected open mode.
-
-#The supported file types.
-from .filetypes.UltimakerFormatPackage import UltimakerFormatPackage
-from .filetypes.GCodeFile import GCodeFile
-from .filetypes.GCodeGzFile import GCodeGzFile
 
 extension_to_mime = {
-    ".ufp": "application/x-ufp",
```

Comment:
Removing the ufp format from the VirtualFile causes the printer no longer being able to read ufp files. Unintentional regression?

Reply:
That should be fixed indeed. I'll do that now.